internal/cli/atlas/deployments/podman/client.go

```diff
@@ -167,7 +167,7 @@ func (o *client) RunContainer(ctx context.Context, opts RunContainerOpts) ([]byt
 	}
 
 	for hostPort, containerPort := range opts.Ports {
-		arg = append(arg, "-p", strconv.Itoa(hostPort)+":"+strconv.Itoa(containerPort))
+		arg = append(arg, "-p", "127.0.0.1:"+strconv.Itoa(hostPort)+":"+strconv.Itoa(containerPort))
```

Comment:
when checking for port availability we're also checking only 127.0.0.1 (localhost)